geowebcache/rest/src/main/java/org/geowebcache/rest/controller/SeedController.java

```diff
@@ -75,8 +75,13 @@ public ResponseEntity<?> doGet(HttpServletRequest req, @PathVariable String laye
         return seedService.getRunningLayerTasks(req, layer);
     }
 
+    @RequestMapping(value = "/seed/{layer}.xml", method = RequestMethod.GET)
```

Comment:
Does this also support layers with dots in their name, or should the path be `"/seed/{layer:.+}.xml"`

Reply:
... and if so, the json endpoint above probably needs the same change too.